spec/acceptance/candlepin_spec.rb

```diff
@@ -21,7 +21,7 @@
         ensure => present,
       }
 
-      ['/usr/share/tomcat/conf', '/etc/candlepin/certs/amqp'].each |$dir| {
+      ['/usr/share/tomcat/conf'].each |$dir| {
```

Comment:
I think we can simplify this a bit since a loop with a single item isn't all that useful :)